str/trtools/merge_str_runner.py

```diff
@@ -42,7 +41,7 @@ def main(
     # Initialise TRTools job to run mergeSTR
     trtools_job = b.new_job(name='mergeSTR')
     trtools_job.image(TRTOOLS_IMAGE)
-    trtools_job.cpu(8)
+    trtools_job.cpu(16)
     # mount using cloudfuse for reading input files
     trtools_job.cloudfuse(f'cpg-{dataset}-main-analysis', '/vcffuse')
```

Comment:
the more robust way to grab the analysis bucket name would be `config.storage.DATASET.analysis` - if we ever change storage patterns (or if you want to run on GCP & Azure without code changes) the config would be the best place to draw from. 

This is fine, wouldn't bother changing here, just an FYI